### ISSUE TITLE
Print the "invalid value" for `output` in CLI flags

### ISSUE THREAD

**Issue by ha-douken** [GoogleChrome/lighthouse#12787]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://www.google.com

### What happened?

_$ npx lighthouse --output=json --output=html https://www.google.com_
throws
_Invalid values. Argument 'output' must be an array from choices "json", "html", "csv"_

node: v12.22.3
"lighthouse": "8.1.0"

### What did you expect?

It to run.

### How were you running Lighthouse?

CLI

### Lighthouse Version

8.1.0

### Node Version

12.22.3

### Relevant log output

```shell
Invalid values. Argument 'output' must be an array from choices "json", "html", "csv"
```

**Comment by patrickhulce** [comment]
Thanks for filing @ha-douken! This is unfortunately, WAI. `--output` is flexible so it allows the use of spaces in between arguments. You need to run `npx lighthouse https://www.google.com --output=json --output=html`. As a rule, always put the URL first.

The error message could definitely be improved though, we'll add the unexpected value to the debug output.

**Comment by svkrclg** [comment]
Is this issue being taken up for dev? If not, shall I pick @patrickhulce ?

**Comment by patrickhulce** [comment]
Go for it @svkrclg!

Basically in https://github.com/GoogleChrome/lighthouse/blob/233c56ec5cc95fdbd0db32885741732da2e74e3e/lighthouse-cli/cli-flags.js#L364-L374

you'll want to add the value that failed validation to our error message instead of just a generic one :)

**Comment by rohan-kulkarni-25** [comment]
Hey @patrickhulce Is this issue solved or needs any more help !! Please let me know I will be happy to do

**Comment by patrickhulce** [comment]
This has been solved by #12836, thanks @rohan-kulkarni-25!
